### DISCUSSION TITLE
Active JSONPath filter not obvious

### DISCUSSION THREAD

**Discussion by IanBurganMod** [Kong/insomnia#3549] [Ideas]
After filtering a response body with JSONPath, the filter remains on the response indefinitely. This is good, desired behavior, but when looking at a request that you may not have looked at for a while, the fact that it is filtered is not clear. For example, a developer might write `$[0:3]`, not notice later, and be confused why there are only 3 items in the response. It is not immediately obvious that a JSONPath filter has been applied. 

A possible solution would be to indicate that a filter has been applied by showing `...` at the top of the response. Alternatively, it could be possible to indicate that a filter has been applied by highlighting the filter bar at the bottom with some color to show the filter is active. Really any indication that a filter has been applied other than the currently subtle change in font color at the bottom would be great.

**Comment by jeremycook** [comment]
This just happened to me. I wrote about it here https://github.com/Kong/insomnia/issues/2968#issuecomment-896340303

**Reply by dimitropoulos** [reply]
Thanks @jeremycook.  We're trying to spend a little more time on these kinds of things so I hope that we can get to this sooner rather than later.

**Reply by jeremycook** [reply]
Cool thanks. I wouldn't want addressing one to slow down addressing the other but it seems like implementing a feature like https://github.com/Kong/insomnia/discussions/3851 in an obvious way could simultaneously address this. I also like that the JSONPath filter is sticks around (except when I forget and don't notice that it has stuck around :)
